src/features/mm_bone_data_feature.cpp

```diff
@@ -97,6 +97,35 @@ BoneState MMBoneDataFeature::_sample_bone_state(Ref<Animation> p_animation, doub
     // Multiply the bone transform by the parent bone transform
     // Keep doing this until the parent bone is the root bone
     // Return the final transform
+    int32_t current_bone_index = _skeleton->find_bone(p_bone_path);
+    String current_bone;
+
+    while (current_bone_index != -1) {
```

Comment:
is there not get global pose?

Reply:
There is, but it returns the global pose of a given bone of the skeleton. I need the global pose of the bone at a given time in the animation, and there's currently no way of doing that.

Reply:
I use it in ik, what do you mean?

Reply:
I looked into my code and I used a cache mechanism on get pose.

https://github.com/V-Sekai/many_bone_ik/blob/main/src/ik_bone_3d.cpp#L57

https://github.com/V-Sekai/many_bone_ik/blob/main/src/math/ik_node_3d.cpp